src/drive/targets/public/index.jsx

```diff
@@ -112,7 +112,8 @@ const init = async () => {
             ) : (
               <Router history={hashHistory}>
                 <Route component={PublicLayout}>
-                  <Route path="files(/:folderId)" component={LightFolderView}>
+                  <Redirect from="/files/:folderId" to="/folder/:folderId" />
+                  <Route path="folder(/:folderId)" component={LightFolderView}>
```

Comment:
Did you check that we have nothing that is based on the previous `files/` url? In some files, I know we do few things with the url.

Reply:
You might be thinking of https://github.com/cozy/cozy-drive/blame/public-nav/src/drive/web/modules/public/Actions.jsx#L46 ? In that case it's the /folder/:id/file/:id sub-route, and it's ok with that change.

I don't think there's more, but in any case the `Redirect` should take care of everything.

Reply:
I was more thinking about the FileExplorer file https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/navigation/FileExplorer.jsx but may be we are not using it here.

Reply:
Indeed, this component isn't used on the public view.